speed-dreams/source-2.2.3/src/simulatedDrivingAssistance/backendUnitTests/SQLDatabaseStorageTests.cpp

```diff
@@ -172,6 +172,60 @@ TEST(SQLDatabaseStorageTests, TestRemoteCorrectFakeCert)
     ASSERT_NO_THROW(sqlDatabaseStorage.Run("test_file.txt", TEST_DATA_DIRECTORY "remote" OS_SEPARATOR "correctRemote"));
 }
 
+/// @brief Tests if the first 4 bytes in buffer file are correctly overwritten by SaveTrialIdToMetadata
+TEST(SQLDatabaseStorageTests, SaveTrialIdTest)
+{
+    // delete buffer file to make sure it doesn't exist
+    filesystem::path bufferPath = filesystem::temp_directory_path();
+    bufferPath.append("sda_blackbox_internal_metadata_buffer.bin");
```

Comment:
Replace this once you moved it to `ConfigEnums.h`. This makes sure that there can never be a miss-match between these two hardcoded filenames.
```suggestion
    bufferPath.append(META_BUFFER_FILENAME);
```